src/Chapter2.hs

```diff
@@ -336,7 +337,12 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList _ _ [] = []
+subList a b (x:xs) | b < a = []
+                    | a < 0 = []
+                    | a > 0 = subList (a-1) (b-1) xs
+                    | b >= 0 = x : subList a (b - 1) xs
+                    | otherwise = []
```

Comment:
That is an interesting idea of the implementation through the recursion 🎨 

However, you could use the standard functions `take` and `drop` to solve this problem as well 🙂 
You would still need to make the checks on negative indexes and if `b < a` as you rightly did in here 👍🏼 

Additional side note: in Haskell, the operators are usually separated from variables with spaces: `a - 1` instead of `a-1` 🙂